backend/FileSystem/FileManager.php

```diff
@@ -42,6 +43,23 @@ public function create(string $relativePath, string $filename = "", mixed $conte
         }
     }
 
+    public function appendNewLine(string $relativePath, string $filename, string $content = ""): void
+    {
+        if ($filename === "") {
+            throw new Exception(
+                "Cannot append to file with no name."
+            );
```

Comment:
Maybe reformat it to one line?

Reply:
I want to implement custom exceptions to store messages away from main classes, but I'll do that for now.